backup/setup/systems.json

```diff
@@ -18,7 +18,8 @@
 		"_id": {
 			"$oid": "0000d186816abba584714c92"
 		},
-		"type": "iserv",
+		"alias": "Zentrales IServ",
+		"type": "oauthIserv",
```

Comment:
For the oauth system(s), of which just 1 exist at the moment and there might never be more in this constellation, we don't use the type, but the oauthConfig. If an oauthConfig exist in the used system, we know that the type /strategy will be oauth. For further clarification, there is a provider field in the oauthConfig which has a value corresponiding to the oauth provider that system is connected to. In this case it's iserv. That's why the "Login via school server" button leads there. With another value in the provider field (and the appropriate configuration in the code) this would be another button an/or would lead to the login page of another oauth provider.